modules/features/podcasts/src/main/java/au/com/shiftyjelly/pocketcasts/podcasts/view/ProfileEpisodeListFragment.kt

```diff
@@ -425,13 +432,19 @@ class ProfileEpisodeListFragment : BaseFragment(), Toolbar.OnMenuItemClickListen
                         CallOnce {
                             analyticsTracker.track(AnalyticsEvent.FREE_UP_SPACE_BANNER_SHOWN)
                         }
-                        ManageDownloadsCard(
-                            totalDownloadSize = downloadedEpisodesSize,
-                            onManageDownloadsClick = {
-                                analyticsTracker.track(AnalyticsEvent.FREE_UP_SPACE_MANAGE_DOWNLOADS_TAPPED, mapOf("source" to SourceView.DOWNLOADS.analyticsValue))
-                                showFragment(ManualCleanupFragment.newInstance())
-                            },
-                        )
+                        Box(
+                            modifier = Modifier
+                                .background(color = MaterialTheme.theme.colors.primaryUi02)
+                                .padding(start = 16.dp, end = 16.dp, top = 16.dp),
```

Comment:
I have kept the padding out of the component because it seems like it should be the parents responsibility. Let me know if you think that is wrong.